.github/workflows/ci.yaml

```diff
@@ -393,17 +393,21 @@ jobs:
         swap-storage: true
         docker-images: true
 
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+    - name: Setup cmake
+      uses: jwlawson/actions-setup-cmake@v1.9
+      with:
+        cmake-version: 3.20.0
```

Comment:
I don't know anything about which version is the right answer, but 3.28 is being used by ONNX RT.  Could cmake be moved to a common version?